pkg/virtualization/core/virtualsystem/virtualmachine.go

```diff
@@ -218,6 +218,33 @@ func (vm *VirtualMachine) Start() error {
 	return vm.WaitForState(Running, StateChangeTimeoutSeconds)
 }
 
+// Resume Virtual Machine
+func (vm *VirtualMachine) Resume() error {
```

Comment:
It is technically possible to just "Start" VMs from Paused or Saved state, but this is extremely simple to add and probably better practice.

Reply:
But it could also create confusion. Let's add some comments here and wmi code